tests/solana.rs

```diff
@@ -999,14 +1000,15 @@ impl Pubkey {
     }
 }
 
+// TODO: This struct is wrong
```

Comment:
The comment could state what's wrong about it too

Reply:
The struct is correct now. The comment was just to remind me to fix it later. I should have removed it.